modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

```diff
@@ -2,7 +2,7 @@
 [id="configuring-namespace-strategies_{context}"]
 = Configuring {orch-namespace} strategies
```

Comment:
Is "namespace strategies" still good header for it? We now have just one strategy. Wouldn't something like "Configuring workspace target namespace" be more appropriate ?